search-commons/src/test/resources/datasource_tickets.json

```diff
@@ -199,8 +203,14 @@
       "username": "1492596@20754.0.0.0"
     },
     "organization": {
-      "id": "https://api.dev.nva.aws.unit.no/cristin/organization/20754.0.0.0",
-      "partOf": []
+      "id": "https://api.dev.nva.aws.unit.no/cristin/organization/20754.3.1.0",
+      "identifier": "20754.3.1.0",
+      "partOf": [
+        "https://api.dev.nva.aws.unit.no/cristin/organization/20754.0.0.0",
+        "20754.0.0.0",
+        "https://api.dev.nva.aws.unit.no/cristin/organization/20754.3.0.0",
+        "20754.3.0.0"
+      ]
```

Comment:
Should these pairs be in separate objects?

Reply:
this is just a list of ids/identifiers, sorting of items has no effect on search results